src/Providers/Maps/Leaflet/Features/Shape.ts

```diff
@@ -0,0 +1,160 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace Provider.Maps.Leaflet.Feature {
+    export class Shape
+        implements
+            OSFramework.Maps.Feature.IShape,
+            OSFramework.Maps.Interface.IBuilder,
+            OSFramework.Maps.Interface.IDisposable
+    {
+        private _isInsideShape: boolean;
+        private _map: OSFramework.Maps.OSMap.IMap;
+        private _markerCoordinates;
```

Comment:
If we can't use any existing type, then I guess we should create one. Check what Gabriel did on Grid, he had to create a bunch of types there recently

Reply:
It is an L.LatLng type. To implement that we need to convert the values from the structure that comes from platform. This is fixed.